src/ai/ai.ts

```diff
@@ -5,6 +5,33 @@ import * as fs from 'fs'
 import * as path from 'path'
 import { InferenceConfig } from '../config.js'
 
+/**
+ * Sanitizes content for safe logging by truncating and removing potential sensitive data
+ * @param content Content to sanitize
+ * @param maxLength Maximum length for logged content
+ * @returns Sanitized content safe for logging
+ */
+function sanitizeForLogging(content: string, maxLength: number = 200): string {
```

Comment:
Please export and use this for testing, and not a copy.

Maybe also move into a `src/utils.ts` file so it can be used in different palces.